conftest.py

```diff
@@ -79,15 +86,14 @@ def appliance_police():
         port_results = {pn: net_check(pp) for pn, pp in ports.items()}
         for port, result in port_results.items():
             if not result:
-                raise Exception('Port {} was not contactable'.format(port), port)
+                raise _AppliancePoliceException('Port {} was not contactable'.format(port), port)
         status_code = requests.get(store.current_appliance.url, verify=False,
                                    timeout=60).status_code
         if status_code != 200:
-            raise Exception('Status code was {}, should be 200'.format(status_code), port)
-    except Exception as e:
-
-        port = e.args[1] if len(e.args) == 2 else None
-        if port == 443:
+            raise _AppliancePoliceException('Status code was {}, should be 200'.format(
+                status_code), port)
+    except _AppliancePoliceException as e:
```

Comment:
This is great, except, it still doesn't handle the possibility that we get a different error during the ```try:```